components/notifications.js

```diff
@@ -566,3 +597,153 @@ function CommentsFlatSkeleton () {
     </div>
   )
 }
+
+const NotificationContext = createContext()
+
+function loadNotifications (storageKey) {
+  const stored = window.localStorage.getItem(storageKey)
+  if (!stored) return []
+  const filtered = JSON.parse(stored).filter(({ sortTime }) => {
+    // only keep notifications younger than 24 hours
+    return new Date(sortTime) >= datePivot(new Date(), { hours: -24 })
+  })
```

Comment:
> **Only keep notifications younger than 24 hours**
>
> This helps with not bloating /notifications and local storage.
>
> It also is a workaround for having to come up with some complicated pagination algorithm which considers how many / which client notifications should be included on each page.

-- 5ff6210e4ffa6d20b64918b822a9ad81f0ae0c10

_I think this is smart lazy stuff_

Reply:
This is a reasonable shim for sure